Twig/Extension/StfalconTinymceExtension.php

```diff
@@ -163,7 +169,9 @@ public function tinymceInit($options = array())
             ),
             'include_jquery' => $config['include_jquery'],
             'tinymce_jquery' => $config['tinymce_jquery'],
+            'asset_package_name' => $asset_package_name,
             'base_url'       => $this->baseUrl
+
```

Comment:
Please delete this blank line and align all elements of the array.
I think it was described in [PSR-2](http://www.php-fig.org/psr/psr-2/)